bittide-instances/src/Bittide/Instances/Hitl/Transceivers.hs

```diff
@@ -84,14 +123,43 @@ goTransceiversUpTest refClk sysClk rst rxns rxps miso =
   -- Transceiver setup
   gthAllReset = unsafeFromActiveLow spiDone
 
+  -- Send counters
+  counters =
+    zipWith3
+      counter
+      transceivers.txClocks
+      transceivers.txResets
+      transceivers.txSamplings
+
+  expectCounterError =
+    zipWith3
+      expectCounter
+      transceivers.rxClocks
+      transceivers.rxResets
+      transceivers.rxDatas
```

Comment:
This looks redundant.

Reply:
Not sure what to do about it.

Reply:
Me neither. I initially thought there may be an easy improvement, but that doesn't seem to be the case.